clients/egui/src/lib.rs

```diff
@@ -261,6 +261,8 @@ impl WgpuLockbook {
             alpha_mode: CompositeAlphaMode::Auto,
             view_formats: vec![],
         };
-        self.surface.configure(&self.device, &surface_config);
+        if surface_config.width * surface_config.height != 0 {
```

Comment:
fixes crash on minimize, pattern I found in other open source projects

Reply:
<img width="900" alt="image" src="https://github.com/lockbook/lockbook/assets/821972/b808a61d-cb6f-4839-aab6-d97676470e2f">